apps/website/src/pages/_global.astro

```diff
@@ -81,10 +81,10 @@ const { content = {} } = Astro.props;
         font-family: var(--font-mono);
       }
 
-      :where(ul:not(.content-wrapper *)) {
-        list-style: none;
-        padding: 0;
-      }
+      // :where(ul:not(.content-wrapper *)) {
+      //   list-style: none;
+      //   padding: 0;
+      // }
```

Comment:
I just commented this out to make the markdown bullet points in the search's response appear correctly. Not sure if this breaks anything